lib/Controllers/list.js

```diff
@@ -61,14 +61,23 @@ List.prototype.fetch = function(req, res, context) {
     var searchOperator = this.resource.search.operator || '$like';
     var searchAttributes =
       this.resource.search.attributes || Object.keys(model.rawAttributes);
+    var stringOperators = {
```

Comment:
likely more performant to add the regex as a method on the controller so it precompiles it for subsequent runs as well